hack/generate-manifest-windows.sh

```diff
@@ -95,49 +90,38 @@ fi
 
 THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-source $THIS_DIR/verify-kustomize.sh
+source $THIS_DIR/verify-helm.sh
 
-if [ -z "$KUSTOMIZE" ]; then
-    KUSTOMIZE="$(verify_kustomize)"
-elif ! $KUSTOMIZE version > /dev/null 2>&1; then
-    echoerr "$KUSTOMIZE does not appear to be a valid kustomize binary"
+if [ -z "$HELM" ]; then
+    HELM="$(verify_helm)"
+elif ! $HELM version > /dev/null 2>&1; then
+    echoerr "$HELM does not appear to be a valid helm binary"
     print_help
     exit 1
 fi
 
-KUSTOMIZATION_DIR=$THIS_DIR/../build/yamls/windows
-
-TMP_DIR=$(mktemp -d $KUSTOMIZATION_DIR/overlays.XXXXXXXX)
+TMP_DIR=$(mktemp -d $THIS_DIR/../build/yamls/chart-values.XXXXXXXX)
```

Comment:
The script is currently not using this directory, as we don not create files for Helm values.
So we can skip creating this directory, and remove the corresponding `rm` command at the end of the script.